updatecli/policies/file/values.yaml

```
@@ -0,0 +1,24 @@
+---
+# Values.yaml contains settings that be used from Updatecli manifest.
+sourceFiles: []
+targetFiles: []
```

```suggestion
targetFiles: []
```

I am wondering if having two different arrays is the right way to do it.
Would it make sense to have a map<src>: dst
And the sourceid could be base64 encoded such 

```
sources:
  "{{ b64enc .srcFile }}":
```